ui/core/sim_manager.ts

```diff
@@ -75,30 +94,34 @@ export class SimManager {
 
 	/**
 	 * Trigger abort for all registered request ids.
-	 * @param type Limit to specific types of requests or all requests.
+	 * @param typeMask Limit to specific types of requests or all requests.
 	 */
-	async abortAll(type: RequestType) {
+	async abortAll(typeMask: RequestTypes) {
 		for (const [id, cfg] of this.running) {
-			if ((cfg.type & type) == 0) continue;
+			if (!(cfg.type & typeMask)) continue;
 			if (cfg.signals) {
-				cfg.signals.abort.trigger();
+				await cfg.signals.abort.trigger();
 			} else {
 				await this.workerPool.abortById(id);
 			}
 		}
 	}
 
-	/** Trigger abort for specific id, if it is registered. */
-	async abortId(id: string): Promise<AbortResponse> {
-		const cfg = this.running.get(id);
+	/**
+	 * Trigger abort for a specific request.
+	 * @param requestId The request id of the request to abort.
+	 * @returns The AbortResponse.
+	 */
+	async abortId(requestId: string): Promise<AbortResponse> {
+		const cfg = this.running.get(requestId);
 		if (cfg) {
 			if (cfg.signals) {
-				cfg.signals.abort.trigger();
-				return AbortResponse.create({id, wasTriggered: true});
+				await cfg.signals.abort.trigger();
+				return AbortResponse.create({requestId, wasTriggered: true});
```

Comment:
Can we run formatting over these files? Or if you use VSCode install the `Prettier` and `ESLint` extension.

Reply:
Doesn't do it on save for me. But did it manually for that file and worker_pool now.